### ISSUE TITLE
Can i differentiate Edge (Legacy) and Edge (Chromium)

### ISSUE THREAD

**Issue by henricook** [matomo-org/device-detector#6598]
Hi all,

In my Matomo install I see:

![image](https://user-images.githubusercontent.com/82004/105509736-bcc0d980-5cc5-11eb-8af6-8b9474b1ba7f.png)

Do you know if it's possible to differentiate Legacy and Chromium Edge?

**Comment by sanchezzzhak** [comment]
Currently, all Edge are grouped as a single browser `Microsoft Edge`

By version, if available, somewhere from the 69th version of Edge uses Chromium V8

**Comment by henricook** [comment]
Yeah i'm not sure I can split by version which makes things pretty difficult. e.g. I want to know if I can end support for legacy edge and the first question I'd ask is "how many of my visitors still use it?"

**Comment by robertmclaws** [comment]
I just used this app to parse an Edge UA string, and the match is still reporting Blink as the engine. Can the regexes be updated so that after version 69 it returns Chromium as the engine? Thanks!
